src/browser_action.js

```diff
@@ -403,7 +403,7 @@ browseraction.createEventDiv_ = function(event) {
     $('<a>').attr({
       'href': 'https://maps.google.com?q=' + encodeURIComponent(event.location),
       'target': '_blank'
-    }).append('<span>' + event.location + '</span>').append($('<img>').addClass('location-icon').attr({
+    }).append('<span>' + event.location + '</span>').addClass('event-location').append($('<img>').addClass('location-icon').attr({
```

Comment:
}).append('<span>').text(event.location).addClass('event-location').append($('<img>').addClass('location-icon').attr({…

.text() will set innerHtml, which inherently does a tiny bit of sanitization.